components/layout/DocsLayout.js

```diff
@@ -13,6 +14,17 @@ import ArrowRight from '../icons/ArrowRight'
 import AnnouncementRemainingDays from '../campaigns/AnnouncementRamainingDays'
 import AnnouncementHero from '../campaigns/AnnoucementHero'
 
+function generateEditLink(post) {
+  if (post.slug.includes('/specifications/')) {
+    if(post.slug.endsWith('/latest')) {
+      return <a href={`https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md`} className="ml-1 underline">Edit this page on Github</a>
+    }
+    return <a href={`https://github.com/asyncapi/spec/blob/${basename(post.slug)}/spec/asyncapi.md`} className="ml-1 underline">Edit this page on Github</a>
+  } else {
+    return <a href={`https://github.com/asyncapi/website/blob/master/pages${post.isIndex ? post.slug + '/index' : post.slug}.md`} className="ml-1 underline">Edit this page on Github</a>
```

Comment:
Please add to links `target="_blank"`, then browser will open page in another tab :)

Reply:
And also add `rel="noopener noreferrer"` :) 

https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
Done.